src/styles/global.css

```
@@ -72,6 +72,9 @@ h3,
   font-size: var(--type-h2);
   font-weight: 400;
   line-height: var(--leading-h2);
+  /* Gochi Hand ships one weight, so the stroke supplies the bulk; em
+     keeps it proportional to the mark's size. */
+  -webkit-text-stroke: 0.04em currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)